pkg/logging/logging.go

```diff
@@ -34,8 +39,27 @@ var (
 	// Logger is the base logger used by Crossplane. It delegates to another
 	// logr.Logger. You *must* call SetLogger to get any actual logging.
 	Logger = logger.WithName("crossplane")
+
+	// ZapLogger is a Logger implementation.
+	ZapLogger = zapLogger
 )
 
+// ZapLogger is a Logger implementation from controller runtime.
+// If development is true, a Zap development config will be used
+// (stacktraces on warnings, no sampling), otherwise a Zap production
+// config will be used (stacktraces on errors, sampling).
+// If disableStacktrace is true, stacktraces enabled on fatals
+// independent of config.
+func zapLogger(development bool, disableStacktrace bool) logr.Logger {
```

Comment:
as discussed in https://github.com/crossplaneio/stack-gcp/pull/49, I like the idea of having a common reusable way to get a logger that only does stack traces for fatal level.  Can we continue pursuing that on this PR?  If we finish and merge this one, then all consumer repos (main crossplane, GCP/AWS/Azure stacks) can just call this one single implementation.